projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -64,6 +76,13 @@ export class TabParagraphContainerComponent
     this.activeTabNum = tabNum;
   }
 
+  ngOnInit(): void {
+    const activeTab = this.activatedRoute.snapshot.queryParams['activeTab'];
```

Comment:
I see a few things here.
1. We can keep the queryParam but when you are changing between the tabs I would expect it to update. An issue now is that if I go back and change tab and refresh I goes to the wrong tab.
2. I don't know if we actually want the query params on the PDP
3. We could avoid the use of the queryParam by passing some data in the route and listening to it in the component. => You can see `update-email.component.ts` line 56